unox.py

```diff
@@ -20,14 +20,20 @@
 
 import sys
 import os
-import time
 import fsevents
-import urllib
 import traceback
 import signal
 
+# Import depending on python version
+if sys.version_info.major < 3:
+    from urllib import quote, unquote
+else:
+    from urllib.parse import quote, unquote
```

Comment:
Have you tested that the script still works on both python2 and python3 with this change?